src/Umbraco.Headless.Client.Net/Delivery/ContentDeliveryEndpoints.cs

```diff
@@ -65,6 +65,12 @@ interface TypedContentDeliveryEndpoints<T> where T : Delivery.Models.IContent
         [Get("/content/url?url={url}&depth={depth}&contentType={contentType}&hyperlinks=false")]
         Task<ApiResponse<T>> GetByUrl([Header(Constants.Headers.ProjectAlias)] string projectAlias, [Header(Constants.Headers.AcceptLanguage)] string culture, string url, string contentType, int depth);
     }
+    [Headers(Constants.Headers.ApiVersion + ":2.3")]
```

Comment:
Looks like you can just use `Constants.Headers.ApiVersionHeader` to get the correctly formatted header for 2.3 if we keep the `ApiVersion` constant as 2.3-

Reply:
And then again....next time we change the ApiVersion it would result in this header being changed 🙃 
Probably good to be explicit here. So no change 🙈